docs/api/duckdb/multi-database-support.md

```diff
@@ -0,0 +1,157 @@
+# Connect DuckDB to other Databases: 
+
+Mosaic utilizes DuckDB, which [can supports multiple databases](https://duckdb.org/2024/01/26/multi-database-support-in-duckdb.html) such as MySQL, PostgreSQL, and SQLite, by using [DuckDB extensions](https://duckdb.org/docs/extensions/overview.html). This capability allows data to be seamlessly read into DuckDB and transferred between systems, enhancing flexibility and interoperability in data management.
+
+## Examples:
+### Connecting to `PostgreSQL`:
+
+``` js
+import { vg, clear } from "http://localhost:5173/dev/setup.js";
+
+clear();
+
+const password = "YOUR_PASSWORD";
+
+const postgres_connection_string = `dbname=postgres user=postgres password=${password} host=127.0.0.1`;
+
+await vg.coordinator().exec([
+  "INSTALL postgres;",   // Install the postgres extension 
+  "LOAD postgres;",   // Load postgres extension manually, 
```

Comment:
The `postgres` extension is one of the [official extensions and is listed as autoloadable](https://duckdb.org/docs/extensions/official_extensions). Are the `INSTALL` and `LOAD` statements actually needed here?